src/engine_core/base/src/Common.hpp

```diff
@@ -0,0 +1,26 @@
+/*! \file Common.hpp
+    \author Alan Ramirez
+    \date 2024-10-13
+    \brief Common definitions
+*/
+
+#pragma once
+
+#include "Platform.hpp"
+
+// By default export the libs.
+#ifndef HUSH_EXPORT_LIB

```

Comment:
Woops, I left this here.
I plan to add support for loadable modules in the future. 
This is not relevant for this PR.